### DISCUSSION TITLE
why can't i use copilot free for students?

### DISCUSSION THREAD

**Discussion by alu0101326351** [community/community#42203] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I used the free two months of GitHub Copilot in a personal account and now I want to use Copilot in my university student account but when I go to the copilot section I get that I have already used my two-month trial along with a message that says I am eligible to use it for free. What I can do?
![Captura de pantalla (342)](https://user-images.githubusercontent.com/72520877/208679233-e319a8a8-f1c4-4ac1-a439-47f33e5f60f0.png)

**Comment by iamcjwilliams** [comment]
@alu0101326351 Sorry for the trouble here, please visit [here](https://education.github.com/discount_requests/pack_application) in order to signup for free student access.

**Comment by Qiming-Liu** [comment]
![image](https://user-images.githubusercontent.com/68600416/211203670-67a19219-62ac-43f0-b26f-21c20d99112a.png)
![image](https://user-images.githubusercontent.com/68600416/211203680-82dea52a-f5b2-44ce-8231-f2f2a3e39dee.png)

Same issue here now

**Reply by iamcjwilliams** [reply]
@Qiming-Liu We apologize for the temporary technical issue on our end that may have caused authentication failures. If you are a verified student please retry logging into your IDE plugin, and Copilot should work as expected. If you run into any further issues, please reply back in detail with any useful screenshots.

**Reply by fahim9778** [reply]
> @Qiming-Liu We apologize for the temporary technical issue on our end that may have caused authentication failures. If you are a verified student please retry logging into your IDE plugin, and Copilot should work as expected. If you run into any further issues, please reply back in detail with any useful screenshots.

Hello @iamcjwilliams,
I followed the processes you mentioned, but I am still getting these error messages consecutively after every 1-2 suggestions, making it totally unusable. This problem persists in `FREE_EDUATIONAL`, `FREE_FACULTY`, and `FREE_OSS_ENGAGED`. 
![Code_qOLk5nwPYP](https://user-images.githubusercontent.com/26436023/211714195-e86ddc14-b935-4b37-b315-7f73cf333fce.png)

Also, as of now, this persists on  my GitHub Billing page. 
![firefox_gMPbiWWVuy](https://user-images.githubusercontent.com/26436023/211714562-9cd42d02-009d-4aaf-bddd-254590e54b9b.png)

**Reply by iamcjwilliams** [reply]
@fahim9778 Refer to this pinned[ post](https://github.com/community/community/discussions/43987)
